recipe/meta.yaml

```diff
@@ -8,31 +8,33 @@ package:
 
 source:
   url: https://github.com/charman2/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 89457f747693a0fdd6c285401af274a2356aab69b3efaa54558d6d92a1cf4ee1
+  sha256: 4147f2da2e7c662970757ba7350dc98a1aeaf72c39c25a531b211dba2e126240
 
 build:
   number: 1
   skip: true  # [py==37]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy 1.24.*                           # [build_platform != target_platform]
+    - python  # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - numpy 1.24.*  # [build_platform != target_platform]
     - cmake
     - make  # [unix]
     - {{ compiler('m2w64-toolchain') }}  # [win]
     - {{ compiler('fortran') }}
     - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - meson =1.3.2
   host:
     - python
-    - setuptools
```

Comment:
Just to confirm, there is no longer any use of `setuptools` in the build?

Reply:
Correct!

Reply:
It looks like CI is running into errors not finding `distutils`, which `setuptools` vendors

Can we try adding this back?

Reply:
No harm in trying, but it shouldn't need it. The main fix in the new version of the package was to migrate from setuptools to meson, since distutils is deprecated

Reply:
For clarity this is the traceback [from CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1045702&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=6d4b912b-175d-51da-0fd9-4d30fe1eb4e7&l=1147 ):

```python
  Traceback (most recent call last):
    File "/home/conda/feedstock_root/build_artifacts/mesas_1727984139908/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/bin/f2py", line 10, in <module>
      sys.exit(main())
    File "/home/conda/feedstock_root/build_artifacts/mesas_1727984139908/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.10/site-packages/numpy/f2py/f2py2e.py", line 691, in main
      run_compile()
    File "/home/conda/feedstock_root/build_artifacts/mesas_1727984139908/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.10/site-packages/numpy/f2py/f2py2e.py", line 619, in run_compile
      from numpy.distutils.system_info import get_info
    File "/home/conda/feedstock_root/build_artifacts/mesas_1727984139908/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.10/site-packages/numpy/distutils/system_info.py", line 196, in <module>
      from numpy.distutils.command.config import config as cmd_config
    File "/home/conda/feedstock_root/build_artifacts/mesas_1727984139908/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.10/site-packages/numpy/distutils/command/config.py", line 19, in <module>
      from numpy.distutils.mingw32ccompiler import generate_manifest
    File "/home/conda/feedstock_root/build_artifacts/mesas_1727984139908/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python3.10/site-packages/numpy/distutils/mingw32ccompiler.py", line 28, in <module>
      from distutils.msvccompiler import get_build_version as get_build_msvc_version
  ModuleNotFoundError: No module named 'distutils.msvccompiler'
```

Reply:
Looks like this issue traces back to `f2py` ( https://github.com/numpy/numpy/issues/24838 ), which was fixed in NumPy 2 ( https://github.com/numpy/numpy/pull/25123 )

Reply:
Have merged the NumPy 2 migrator from PR ( https://github.com/conda-forge/mesas-feedstock/pull/49 ) into this PR. Let's see if that helps

Reply:
In some jobs this appears to be fixed! 🎉  However there are new errors (noted below) 😞 Please take a look at the comments below

We still see this error in a few jobs. So let's keep this thread open

Though `setuptools` isn't helping in any case. So think we can go ahead and drop it again